README.md

```diff
@@ -32,14 +32,15 @@ More information on terminology can be found [here](docs/terminology.md).
 To populate that database
 ```
 ## Initialize DBs
-./phg initdb --dbpath /path/to/dbs
+./phg initdb --db-path /path/to/dbs
```

Comment:
should SetupEnvironment be added prior to this?  I don't see it referenced here.

Reply:
yes it should be.  I really just updated the parameters I changed.  I can add it now though, but anything else will need to be added later.  Updating the Readme perfectly is out of scope for this PR